bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/sitequality/TimingWindowDialog.java

```diff
@@ -0,0 +1,285 @@
+package jsky.app.ot.gemini.editor.sitequality;
+
+import edu.gemini.shared.util.HourMinuteFormat;
+import edu.gemini.shared.util.immutable.*;
+import edu.gemini.spModel.gemini.obscomp.SPSiteQuality.TimingWindow;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.FocusAdapter;
+import java.awt.event.FocusEvent;
+import java.awt.event.KeyAdapter;
+import java.awt.event.KeyEvent;
+import java.text.*;
+import java.util.Date;
+import java.util.TimeZone;
+
+class TimingWindowDialog extends JDialog {
```

Comment:
The moved and heavily modified `TimingWindowDialog` class.
I think it is much cleaner now, and modality works as one would expect.